modules/account-lib/src/coin/cspr/transactionBuilderFactory.ts

```diff
@@ -22,8 +24,19 @@ export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
   }
 
   /** @inheritDoc */
-  from(raw: Uint8Array | string): TransactionBuilder {
-    throw new NotImplementedError('from not implemented');
+  from(raw: DeployUtil.Deploy): TransactionBuilder {
+    this.validateRawTransaction(raw);
+    const tx = new Transaction(this._coinConfig);
+    tx.casperTx = raw;
+
+    if (tx.casperTx.session.tag === DeployTag.transfer) {
+      return this.getTransferBuilder(tx);
+    } else if (tx.casperTx.session.tag === DeployTag.moduleBytes) {
+      // TODO , we have not parameter to absolute determite if is a walletInit contract
```

Comment:
Please add a ticket number
readability-nit:
```suggestion
      // TODO(STLX-XXXX)  identify wallet initialization deploy
```